docs/jwst/introduction.rst

```diff
@@ -451,18 +451,23 @@ local configuration file.
 
 A configuration file should be used when there are parameters a user wishes to
 change from the default/CRDS version for a custom run of the step. To create a
-configuration file with the default values, use the command: ::
+configuration file add ``--save-parameters <filename.asdf`` to the command:
 
- strun <step.class> --save-parameters <filename.asdf>
+ strun <step.class> <required-input-files> --save-parameters <filename.asdf>
 
 For example, to get the parameters for the jump step, use:
```

Comment:
I'm not sure I would characterize this example as the way to just "get the parameters for the jump step". You're doing more than that: you're running the step and having it save the parameters for you. If you just wanted to get the parameters, you could do something like ``stspec jwst.jump.JumpStep``.

Reply:
Modified the documentation to make it clear it was a run of the step.

Though stspec gives the {{spec}} of a step, it currently does not produce anything directly useful in creating parameter files. That may be something to modify such that it would do so.